### PR TITLE
NFS: Investigate `FormFieldBlueprint` to create `FormFieldExtensions` in the Scaffolder Plugin

### PR DESCRIPTION
**Pull request by benjdlambert**
Step one of investigating what the Scaffolder plugin looks like in the new frontend system and the API's that we would need to build. This allows the creation of field extensions using the new `FormFieldBlueprint`, which get collected together in a private `formFieldsApi`.

### CODE REVIEW

File: packages/scaffolder-internal/CHANGELOG.md
Comment (Rugvip) [inline packages/scaffolder-internal/CHANGELOG.md:1]: 🧹

**Review by Rugvip — COMMENTED** [review]
Nice! 😁 just some bits and bobs

File: plugins/scaffolder/package.json [hunk 1]
```diff
@@ -73,6 +73,7 @@
     "@backstage/plugin-scaffolder-common": "workspace:^",
     "@backstage/plugin-scaffolder-react": "workspace:^",
     "@backstage/types": "workspace:^",
+    "@backstage/version-bridge": "workspace:^",
```
[the remark this answers is not in the document]
Comment (benjdlambert) [reply plugins/scaffolder/package.json:76]: The opaque helper brought this in

[the remark this answers is not in the document]
Comment (Rugvip) [reply plugins/scaffolder/package.json:76]: Ah right, but that in turn doesn't need it afaik?

[the remark this answers is not in the document]
Comment (benjdlambert) [reply plugins/scaffolder/package.json:76]: image

[the remark this answers is not in the document]
Comment (benjdlambert) [reply plugins/scaffolder/package.json:76]: Now I gotchu

**Review by Rugvip — APPROVED** [review]
:shipit: 🎉

**Pull request merged**